crates/pallet-vesting/src/api.rs

```diff
@@ -2,9 +2,10 @@
 
 use codec::{Decode, Encode};
 use frame_support::sp_runtime::DispatchError;
+use scale_info::TypeInfo;
 
 /// An error that can occur while evaluating the lock logic.
-#[derive(Debug, Encode, Decode, PartialEq, Eq)]
+#[derive(Debug, Encode, Decode, PartialEq, Eq, TypeInfo)]
```

Comment:
This is something that only appeared after I last viewed the code - why this change?

Reply:
Just checked all commits after your last review. It wasn't added there. It was before.

Reply:
https://github.com/humanode-network/humanode/pull/861#discussion_r1750791085

Reply:
What was changed in this file then? Might be the ` where Id: Encode` change below?